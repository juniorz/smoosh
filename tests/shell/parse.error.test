echo ')' >scr
$TEST_SHELL scr || echo sh ok
{ echo eval ')' | $TEST_SHELL -i ; } || echo eval ok
$TEST_SHELL -c '. ./scr' || echo dot ok
