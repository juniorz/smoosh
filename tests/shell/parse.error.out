sh ok
eval ok
dot ok
